go.mod

```diff
@@ -1,22 +1,43 @@
 module github.com/mattermost/mattermost-plugin-starter-template
 
-go 1.12
+go 1.13
```

Comment:
1.14 ? :)

Reply:
I will do a follow-up PR once I have time to test 1.14. 👍